pandas/tseries/common.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 
-from pandas.types.common import (_NS_DTYPE, _TD_DTYPE,
+from pandas.types.common import (_NS_DTYPE, _TD_DTYPE,  # noqa
```

Comment:
actually I think you can just take these out (you may need to import them where they are actually used though)